lib/logster/rails/railtie.rb

```diff
@@ -14,9 +14,12 @@ def self.set_logger(config)
     store.level = Logger::Severity::WARN if Rails.env.production?
 
     logger = Logster::Logger.new(store)
-    logger.chain(::Rails.logger)
+    original_logger = ::Rails.logger
+    logger.chain(original_logger)
     logger.level = ::Rails.logger.level
-
+    if defined?(ActiveSupport::TaggedLogging) && ActiveSupport::TaggedLogging === original_logger
+      logger = ActiveSupport::TaggedLogging.new(original_logger)
+    end
```

Comment:
note:

1. I haven't tested this specific change myself
2. there are other loggers that use a similar interface, such as [faraday detailed logger ](https://github.com/envylabs/faraday-detailed_logger/blob/a6f0242686004c6fcc7936b0ec3c4a800ab4713a/lib/faraday/detailed_logger/tagged_logging.rb)